## DetectionFull.msg
Header header

# The image containing the detetions
sensor_msgs/Image image

# The array containing all the detections
DetectedArray detections
